src/Control/Middleware/FlushMiddleware.php

```diff
@@ -17,16 +14,10 @@ class FlushMiddleware implements HTTPMiddleware
     public function process(HTTPRequest $request, callable $delegate)
     {
         $kernel = Injector::inst()->get(Kernel::class);
-        if ((method_exists($kernel, 'isFlushed') && $kernel->isFlushed())) {
+        if ($kernel->isFlushed()) {
```

Comment:
That method is part of the `Kernel` interface, so no need for the old method_exists check.